src/assets/kilt_small.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This can be optimized

Reply:
Right, thanks, fixed via this https://github.com/svg/svgo 👍